modules/appeals_api/spec/docs/decision_reviews/higher_level_reviews_spec.rb

```diff
@@ -195,230 +137,117 @@
-          it_behaves_like 'rswag example', desc: 'returns a 200 response', response_wrapper: :raw_body
-        end
+      response '404', 'Veteran not found' do
+        schema '$ref' => '#/components/schemas/errorModel'
 
-        it_behaves_like 'rswag 500 response'
+        let(:benefit_type) { 'compensation' }
+        let(:'X-VA-SSN') { '000000000' }
+        let(:'X-VA-Receipt-Date') { '2019-12-01' }
+
+        it_behaves_like 'rswag example',
+                        cassette: 'caseflow/higher_level_reviews/not_found',
+                        desc: 'Returns a 404 response'
       end
-    end
-  else
-    path '/schemas/{schema_type}' do
-      get 'Gets JSON schema related to Higher-Level Review.' do
-        scopes = AppealsApi::HigherLevelReviews::V0::HigherLevelReviewsController::OAUTH_SCOPES[:GET]
-        tags 'Higher-Level Reviews'
-        operationId 'hlrSchema'
-        description 'Returns the [JSON Schema](https://json-schema.org) related to the `POST /forms/200996` endpoint'
-        security DocHelpers.decision_reviews_security_config(scopes)
-        produces 'application/json'
-
-        examples = {
-          '200996': { value: '200996' },
-          'address': { value: 'address' },
-          'non_blank_string': { value: 'non_blank_string' },
-          'phone': { value: 'phone' },
-          'timezone': { value: 'timezone' }
-        }
 
-        parameter(name: :schema_type,
-                  in: :path,
-                  type: :string,
-                  description: "Schema type. Can be: `#{examples.keys.join('`, `')}`",
-                  required: true,
-                  examples:)
-
-        examples.each do |_, v|
-          response '200', 'The JSON schema for the given `schema_type` parameter' do
-            let(:schema_type) { v[:value] }
-            it_behaves_like 'rswag example', desc: v[:value], extract_desc: true, scopes:
-          end
-        end
+      response '422', 'Bad receipt date' do
+        schema '$ref' => '#/components/schemas/errorModel'
 
-        response '404', '`schema_type` not found' do
-          schema '$ref' => '#/components/schemas/errorModel'
-          let(:schema_type) { 'invalid_schema_type' }
-          it_behaves_like 'rswag example', desc: 'schema type not found', scopes:
-        end
+        let(:benefit_type) { 'compensation' }
+        let(:'X-VA-SSN') { '872958715' }
+        let(:'X-VA-Receipt-Date') { '1900-01-01' }
 
-        it_behaves_like 'rswag 500 response'
+        it_behaves_like 'rswag example',
+                        cassette: 'caseflow/higher_level_reviews/bad_date',
+                        desc: 'Returns a 422 response'
       end
+
+      it_behaves_like 'rswag 500 response'
+
+      response '502', 'Unknown error' do
+        # schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'errors', 'default.json')))
+        # #/errors/0/source is a string 'Appeals Caseflow' instead of an object...
```

Comment:
Not a blocker for this ticket, but if we could spend a few minutes investigating this apparent inconsistency in our error responses, it might be a good thing to fix for v0.

Reply:
Yeah, I tried uncommenting it and investigating, but things were not super clear - definitely want to revisit these a bit in general and talk about updates